### PR TITLE
Eduardo jimenez

### PR DESCRIPTION
**Pull request by l8nightswithJS**
Final project commit for  UI-III-Flexbox.

### CODE REVIEW

File: great-idea/css/index.css [hunk 1]
```diff
@@ -170,30 +172,122 @@ h5 {
 	margin: 0auto;
```
Comment (MisterC-Rogers) [inline great-idea/css/index.css:172]: a better way to do this is 
margin: 0 auto; 
the space makes it easier to read

File: great-idea/css/index.css [hunk 2]
```diff
@@ -170,30 +172,122 @@ h5 {
...
-}
 
 .num-email {
 	margin-top: 10px;
 	align-self: auto;
 }
 
 h6 {
-	margin-top: 50px;
+	margin-top: 100px;
```
Comment (MisterC-Rogers) [inline great-idea/css/index.css:182]: on this you could one line it like this 
margin: 100px 0 85px 0;
the order goes top, right, bottom, left

**Review by MisterC-Rogers — COMMENTED** [review]
other then what I added from a quick look things look great

File: great-idea/css/index.css (hunk 1 above)
Comment (l8nightswithJS) [reply great-idea/css/index.css:172]: Awesome I will remember that, I thought about doing the margin in one line but I wasn't sure if it was the same as padding, glad that it is makes it easy, I should have googled it, thanks for the feedback.

**Pull request merged**